scripts/install_dependencies/install_NEO_OCL_driver.sh

```diff
@@ -31,6 +31,55 @@ UBUNTU_VERSION=
 DISTRO=
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 INSTALL_DRIVER_VERSION='19.41.14441'
+AVAILABLE_DRIVERS=("19.41.14441" "20.35.17767")
+
+
+print_help()
+{
+   # Display Help
+   usage="Usage: $(basename "$0") [OPTIONS]...
+Installs the Graphics Driver for OpenCL on Linux
+
+    Available options:
+    -y                      Replace the currently installed driver with the newer version.
+    -d, --install_driver    Force change version to one of available to install drivers.
```

Comment:
Maybe `Manually select driver version to install (one of ${AVAILABLE_DRIVERS[@]})` would be slightly more clear.